s2n.mk

```diff
@@ -45,6 +45,11 @@ ifneq ($(NO_STACK_PROTECTOR), 1)
 DEFAULT_CFLAGS += -Wstack-protector -fstack-protector-all
 endif
 
+# Define S2N_ENTER_FIPS_MODE - to be used for testing when present.
+ifdef S2N_ENTER_FIPS_MODE
```

Comment:
I think it would be useful to have `TEST` somewhere in the name so it is clear this envar is only for tests